jp2_pc/Source/Lib/EntityDBase/Water.cpp

```diff
@@ -117,7 +117,7 @@
 
 //#include "Config.hpp"
 //#define TARGET_PROCESSOR PROCESSOR_PENTIUM
-//#define VER_ASM			0
+#define VER_ASM			0
 
```

Comment:
`VER_ASM` is actually a global macro from `BuildVer.hpp` and set to `1`, but deactivating it globally does not work yet.